pvactools/tools/pvacview/styling.R

```diff
@@ -20,6 +20,7 @@ callback <- function(hla_count, score_mode) {
     "        'Allele',",
     "        'Pos - The one-based position of the start of the mutation within the epitope sequence. 0 if the start of the mutation is before the epitope (as can occur downstream of frameshift mutations).',",
     "        'Prob Pos - Problematic positions within the best peptide.',",
+    "        'Num Included Peptides - The total number of peptides for this mutation.',",
```

Comment:
This might be a bit confusing for our users. While this does reflect the total number of peptides available in pVACview with meta-data, it's not all of the peptides resulting from the mutation.  In the context of the aggregated report this is described as:
> The number of included peptides according to the `--aggregate-inclusion-binding-threshold` and `--aggregate-inclusion-count-limit`

Here it might make sense to word it as "The number of top-scoring, unique peptides included in this report". @malachig @chrisamiller do you have opinions on this wording.